pkg/util/processinfo.go

```diff
@@ -59,6 +59,7 @@ type ProcessInfo struct {
 	Port                  string
 	ResourceGroupName     string
 	SessionAlias          string
+	RedactSQL             string
```

Comment:
Because of the default value for string is "", and the default behavior for this struct is no redact, new `parser.Normalize` didnt default to `ON` but `OFF`.